xml/System.Security.Cryptography.X509Certificates/X509ChainPolicy.xml

```diff
@@ -666,6 +666,8 @@ The parameterless constructor for the <xref:System.Security.Cryptography.X509Cer
 Calling the <xref:System.Security.Cryptography.X509Certificates.X509ChainPolicy.Reset> method replaces the value of <xref:System.Security.Cryptography.X509Certificates.X509ChainPolicy.VerificationTime> with the current system time, 
 not with the time that the object was constructed.
 
+This property is currently not respected on non-Windows OS-es due to platform limitations.
```

Comment:
I think this is an overly broad statement. `VerificationTime` _does_ work on all platforms with regard to a certificate's notBefore and notAfter. e.g a VerificationTime of 2019-12-30 will not consider a certificate with a notAfter 2020-01-01 expired.

I believe it is only the case of "when was it revoked?" that Windows accounts for VerificationTime while the other platforms do not.

Reply:
Thanks for clarification, issue implied the property doesn't work at all and I haven't used this property myself. I've fixed the note.